offchain-modules/packages/x/src/handlers/eth.ts

```diff
@@ -1,12 +1,14 @@
-import { generateAddress, parseAddress } from '@ckb-lumos/helpers';
+import { parseAddress } from '@ckb-lumos/helpers';
```

Comment:
`generateAddress` was used in `toCKBAddress2021`

Reply:
done.